modules/complete-assisted-installer-oci-temp.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="complete-assisted-installer-oci-temp_{context}"]
+= Completing the remaining {ai-full} steps - 2
+
+After you provision {oci-first} resources and upload {product-title} custom manifest configuration files to {oci}, you must complete the remaining cluster installation steps on the {ai-full} before you can create an instance {oci}. These steps include assigning node roles and adding custom manifests.
+
+.Prerequisites
+
+* You created a resource stack on {oci} that includes the custom manifest configuration files and {oci} Resource Manager configuration resources. See "Provisioning {oci} infrastructure for your cluster".
+
+== Assigning node roles
+
+Following host discovery, the role of all nodes appears as *Auto-assign* by default. Change each of the node roles to either *Control Plane node* or *Worker*.
+
+.Procedure
+
+. From the {ai-full} user interface, go to the *Host discovery* page.
+
+. Under the *Role* column, select either *Control plane node* or *Worker* for each targeted hostname. Then click *Next*.
++
+[IMPORTANT]
+====
+Before continuing to the next step, wait for each node to reach `Ready` status.
+====
+
+. Accept the default settings for the *Storage* and *Networking* pages. Then click *Next*.
+
+== Adding custom manifests
+
+Add the mandatory custom manifests provided by Oracle. For details, see link:https://github.com/dfoster-oracle/oci-openshift/blob/v1.0.0-release-preview/custom_manifests/README.md[Custom Manifests (Oracle documentation).] 
+
+.Procedure
+
+. On the *Custom manifests* page, in the *Folder* field, select `manifests`. This is the {ai-full} folder where you want to save the custom manifest file.
+
+. In the *File name* field, enter a filename, for example, `dynamic_custom_manifest`.
+
+. Paste the contents of the `dynamic_custom_manifest` file that you copied from {oci}:
+
+.. In the *Content* section, click the *Paste content* icon. 
+
+.. Click *OK* to close the dialog box, and then click *Ctrl+V*.  
```

Comment:
I think this step is only for firefox.
On chrome, the click on Paste button should just work.
@ammont82